status-reporter/status_reporter/rabbitmq_reporter.py

```diff
@@ -13,17 +13,103 @@
 #  * See the License for the specific language governing permissions and
 #  * limitations under the License.
 
-from cloudify.cluster_status import CloudifyNodeType
 
-from .status_reporter import Reporter
+from requests import get
+from requests.exceptions import RequestException
+
+from cloudify.cluster_status import (CloudifyNodeType, ServiceStatus,
+                                     NodeServiceStatus)
+
+from .status_reporter import Reporter, logger
+from .constants import STATUS_REPORTER_CONFIG_KEY
+from .utils import get_systemd_services, determine_node_status
+
+NODES_API = 'nodes'
+VHOSTS_API = 'vhosts'
+EXTRA_INFO = 'extra_info'
```

Comment:
well this is shared also for the DB reporter, so please also use that there (ofc from the constants file)